docker/postgres/Dockerfile

```diff
@@ -0,0 +1,16 @@
+ARG POSTGRES_VERSION=16
+
+FROM postgis/postgis:${POSTGRES_VERSION}-3.4
+
+RUN apt-get update && apt-get install -y git build-essential software-properties-common postgresql-server-dev-${POSTGRES_VERSION} clang
+
+RUN git clone https://github.com/citusdata/pg_cron.git
```

Comment:
This command retrieves pg_cron from the main branch, which could potentially be unstable. It's advisable to clone it using a specific tag for more stability.

Alternatively, for a more reliable installation, I strongly suggest using the package manager. You can install it via apt by running `apt-get update && apt-get install -y postgresql-16-cron`. This ensures a smoother and more controlled deployment process.